src/components/RenameHeader.tsx

```diff
@@ -59,7 +59,7 @@ const RenameHeader: FC<Props> = ({
           <ol className="p-breadcrumbs__items">
             {parentItems.map((item, key) => (
               <li
-                className="p-heading--4 u-no-margin--bottom continuous-breadcrumb"
+                className="p-heading--4 u-no-margin--bottom continuous-breadcrumb u-truncate"
```

Comment:
The truncate class should be added to the 2nd item in the header further down in line 102 instead. This here is the parent -- the "Instances" link, which doesn't need truncation.